rust/connlib/libs/common/src/control.rs

```diff
@@ -24,6 +25,10 @@ use url::Url;
 use crate::{get_user_agent, Error, Result};
 
 const CHANNEL_SIZE: usize = 1_000;
+const HEARTBEAT: Duration = Duration::from_secs(30);
+const HEARTBEAT_TIMEOUT: Duration = Duration::from_secs(35);
```

Comment:
Why is this one 35 seconds? Shouldn't the reply to a heartbeat be much quicker? Is there even a reply to a heartbeat?

Reply:
I am asking because I am porting these changes to the `phoenix-channel` crate. I am tracking request IDs there and if we make the timeout _larger_ than the actual heartbeat interval, I need to always keep around two IDs.

Reply:
This is not the response delay but how often I expect a message in the channel, so every 30 seconds there should be a heartbeat and the delay would be at most 5 seconds.

This was easier to implement than having a timer awaiting the response for each heartbeat for this version of the channel but the timeout for each heart beat would be ~5 seconds or we can make it even shorter.

Reply:
Thanks for clarifying!